frontend/components/home/js/home.component.ts

```diff
@@ -368,7 +368,7 @@ class HomeController implements ng.IController {
 		}
 
 		if (!this.topLogo) {
-			this.topLogo = "/images/3drepo-logo-white.png";
+			this.topLogo = "/images/3D-repo-logo-white-shadow.png";
```

Comment:
Any reason for this change? Logo is no longer showing:
![image](https://user-images.githubusercontent.com/11945337/46728172-1d08ea00-cc7a-11e8-918a-2c4111bf3792.png)